corehq/apps/geospatial/tests/test_views.py

```diff
@@ -335,23 +335,25 @@ def tearDownClass(cls):
         super().tearDownClass()
 
     def test_get_users_with_gps(self):
-        expected_output = {
-            'user_data': [
-                {
-                    'id': self.user_a.user_id,
-                    'username': self.user_a.raw_username,
-                    'gps_point': '12.34 45.67',
-                },
-                {
-                    'id': self.user_c.user_id,
-                    'username': self.user_c.raw_username,
-                    'gps_point': '45.67 12.34',
-                },
-            ],
+        expected_results = {
+            self.user_a.user_id: {
+                'id': self.user_a.user_id,
+                'username': self.user_a.raw_username,
+                'gps_point': '12.34 45.67',
+            },
+            self.user_c.user_id: {
+                'id': self.user_c.user_id,
+                'username': self.user_c.raw_username,
+                'gps_point': '45.67 12.34',
+            }
         }
         self.client.login(username=self.username, password=self.password)
         response = self.client.get(self.endpoint)
-        self.assertEqual(response.json(), expected_output)
+        response_json = response.json()
+        self.assertTrue('user_data' in response_json)
```

Comment:
nit: `assertIn` would give richer details if this assertion fails

```suggestion
        self.assertIn('user_data', response_json)
```